cycles-ledger/src/storage.rs

```diff
@@ -1567,25 +1672,55 @@ pub async fn withdraw(
         // subtract the fee to pay for the reimburse block
         let amount_to_reimburse = amount.saturating_sub(config::FEE);
         if amount_to_reimburse.is_zero() {
-            return Err(FailedToWithdraw {
+            return Err(FailedToWithdrawFrom {
```

Comment:
There is no separate error type for the `withdraw` function?
But there is still a `WithdrawError` at the global level, right?

Reply:
Correct. `withdraw` does `.map_err(withdraw_from_error_to_withdraw_error)` at the end